sky/resources.py

```diff
@@ -275,7 +275,35 @@ def _set_region_zone(self, region: Optional[str],
         self._region, self._zone = self._cloud.validate_region_zone(
             region, zone)
 
-    def _try_validate_instance_type(self):
+    def get_valid_region_zones(self) -> List[Tuple[clouds.Region, clouds.Zone]]:
```

Comment:
nit: in return type, should the zones be a List[clouds.Zone]?

Reply:
Good catch. Fixed.